OrbitGl/App.cpp

```diff
@@ -1381,6 +1388,7 @@ void OrbitApp::set_selected_thread_id(ThreadID thread_id) {
 const TextBox* OrbitApp::selected_text_box() const { return data_manager_->selected_text_box(); }
 void OrbitApp::SelectTextBox(const TextBox* text_box) {
   data_manager_->set_selected_text_box(text_box);
+  data_manager_->set_highlighted_function(0);
```

Comment:
Thanks for pointing out the "all timers highlighted issue". I set highlighted_function as 0 to indicate no highlighted function is picked from the live functions data view panel. You remind me that I should also check the function address of selected text box because when text box is selected from places such as Transfer Queue, the function address is 0.

Reply:
There are some Thread Fake ID, like -1 or -2, I don't remember. If you declare a constant kNotUsedThreadID or whatever (with a value of -3 for example), you can set the value to this ID and you can avoid the double check

Reply:
Thanks so much for telling me about the details of Thread Fake ID. But I'm not quite understand what the 'double check' refers to. Do you mean the check of uninterested threads?
BTW, I fixed the "all timers highlighted issue" in the fixup commit a654230 by  checking the function address of selected textbox  when declaring `need_highlight`.

Reply:
Ignore my double check comment. I misunderstood something. Anyway, if you still want to set 0 there, you should create a constant for that.